proto/panacea/datapool/v2/tx.proto

```diff
@@ -15,6 +15,9 @@ service Msg {
   // RegisterDataValidator defines a method for registration of data validator.
   rpc RegisterDataValidator(MsgRegisterDataValidator) returns (MsgRegisterDataValidatorResponse);
 
+  // UpdateDataValidator defines a method for updating of data validator.
+  rpc UpdateDataValidator(MsgUpdateDataValidator) returns (MsgUpdateDataValidatorResponse);
```

Comment:
Good to have this Tx. Thank you.

But, let's not execute this tx automatically in the `panacea-data-market-validator`. If operators accidentally change their `endpoint` value in their `config.toml`, and if tx is executed automatically at that time, they will pay lots of tx fees unintentionally. So, it would be let them execute this tx maunally when they really need.

Then, I have a new concern that letting dataval operators execute the CreateDataValidator manually as well. Then, the dataval process can just check whether it was registered to Panacea. If not, it can print an error and exit.
I didn't think seriously about this topic before, because the dataval accepts a mnemonic anyway to use it for signing DataValidationCerts. But, I'm thinking if executing any txs automatically is degrading its UX. What do you think? @gyuguen @hansol-medi @inchori

Reply:
Are you saying that you run RegisterDataValidator and UpdateDataValidator manually?

If so, I agree.
I think it's better to run this manually via command.
For tx that may cause problems, it seems better to operate manually by the user.

Reply:
I also agree to operate those Txs manually. It may cause problems as gyuguen said, and we couldn't make it fully automatic for all cases as youngjoon concerned.

Reply:
Thanks for discussion!. Register and Update of Data Validator Transactions will be executed manually based on above reason.